Interfaces/MainMenu.xib

```diff
@@ -636,11 +625,6 @@
                                     <action selector="removeDatabase:" target="-1" id="659"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Drop Database..." alternate="YES" id="1207">
-                                <connections>
-                                    <action selector="removeDatabase:" target="-1" id="1208"/>
-                                </connections>
-                            </menuItem>
```

Comment:
Removed drop database menu item?

Reply:
There were two `Remove database` and `Drop database`. Both calling the same function.